src/wolff_producer.erl

```diff
@@ -671,7 +725,7 @@ enqueue_calls(Calls, #{replayq := Q,
                        partition := Partition,
                        config := Config0
                       } = St0) ->
-  {QueueItems, PendingAcks, CallByteSize, NrOfMsgs} =
+  {QueueItems, PendingAcks, CallByteSize, _NrOfMsgs} =
```

Comment:
there is no need to calculate `NrOfMsgs` in the foldl fun any more.

Reply:
:heavy_check_mark: